doc/frameworks/pytorch/using_pytorch.rst

```diff
@@ -200,29 +200,97 @@ fit Optional Arguments
 Distributed PyTorch Training
 ============================
 
-You can run a multi-machine, distributed PyTorch training using the PyTorch Estimator. By default, PyTorch objects will
-submit single-machine training jobs to SageMaker. If you set ``instance_count`` to be greater than one, multi-machine
-training jobs will be launched when ``fit`` is called. When you run multi-machine training, SageMaker will import your
-training script and run it on each host in the cluster.
+SageMaker supports the `PyTorch DistributedDataParallel (DDP)
+<https://pytorch.org/docs/master/generated/torch.nn.parallel.DistributedDataParallel.html>`_
+package. You simply need to check the variables in your distributed training script,
+such as the world size and the rank of the current host,
+to be matching with the specs of the ML instance type you use.
```

Comment:
I am not super clear what this means. I thought world size and rank do not depend on the ML instance type. If they do, we should add more information to explain that.

Reply:
In the `Adapt Your Training Script` section, you need to specify `world_size` as string. This requires users to specify the world size either in the script or the estimator's hyperparameter for arg parser.

Otherwise, you should add a line in the script to use a function like `dist.get_world_size()` to automatically detect.

Refer to the guide we have in the SMDDP documentation: https://docs.aws.amazon.com/sagemaker/latest/dg/data-parallel-modify-sdp-pt.html#data-parallel-how-to-modify-sdp-pt

Reply:
And I wanted to point out the world size and rank assignment will be done based on the total number of GPUs in a cluster of an instance type.

Reply:
Changed the paragraph a bit. Let me know if it sounds better now. Thanks!